resource/out

```diff
@@ -30,7 +30,8 @@ def create_client(src_dir, source={}, params={}):
     client = auth0.management.get_or_create(Client(
         name=source['app-name'],
         callbacks=[f'{app_url}/callback'],
-        allowed_origins=[app_url]))
+        allowed_origins=[app_url],
+        web_origins=[app_url]))
```

Comment:
I quickly looked for some Auth0 documentation regarding this, I didn't find much a part from a person who's [getting a `Additional properties not allowed: web_origins` error](https://community.auth0.com/t/setting-web-origins-when-creating-a-client-using-the-management-api/9130).

[Here](https://community.auth0.com/t/is-it-possible-to-add-allowed-web-origins-with-the-management-api/9385/1) someone suggest to create the client and then `PATCH` it but it's referring to API v2 🤷‍♂️ .

Reply:
interesting - i'll change it to be safe.